src/test/common/router/routablePath.ts

```diff
@@ -34,6 +34,13 @@ describe('RoutablePath', () => {
       expect(new RoutablePath('/case/:id/payment/:payment-type').evaluateUri({ id: '999', 'payment-type': 'card' }))
         .to.be.equal('/case/999/payment/card')
     })
+
+    it('should fail when path parameter placeholder has invalid value', () => {
+      [{ id: undefined }, { id: null }, { id: 'null' }, { id: 'undefined' }, { id: '' }].forEach(invalidValue => {
```

Comment:
null or undefined are valid names and there are people with the last name of null let's not get to smart.
Please remove these checks

Reply:
hmm, since this is only in uri's i'll accept it, if a comment is added explaining it.
i.e. no one should be copy pasting that check elsewhere

Reply:
Added comment to make it more clear.